swagger.yaml

```diff
@@ -1457,13 +1432,41 @@ components:
           format: uuid
           description: User's ID in Keycloak.
         attributes:
-          $ref: "#/components/schemas/Attributes"
-        severaUserId:
-          type: string
-          description: The Severa user ID associated with the Keycloak attribute.
-        severaKeyword:
+          $ref: "#/components/schemas/Attributes" 
+
+    UpdateUserAttributeResponse:
```

Comment:
You are missing the "message" attribute here which you are returning form the lambda. I would suggest just removing the message from the lambda as we don't typically return a message. So change required in the spec but message needs to be removed from the lambda updateUserAttributeHandler return.